staging/src/k8s.io/apiextensions-apiserver/test/integration/basic_test.go

```diff
@@ -194,6 +194,29 @@ func testSimpleCRUD(t *testing.T, ns string, noxuDefinition *apiextensionsv1beta
 					t.Errorf("expected %v, got %v", e, a)
 				}
 			}
+
+			// Update test
+			gottenNoxuInstance.Object["updated"] = true
```

Comment:
`unstructured.SetNestedField(gottenNoxuInstance.Object, true, "updated")`?

Reply:
@sttts looks like this will fix the panic :)

Reply:
or check for `disabledVersions[version2]`. Object is nil in that case and it panics.

Reply:
just noticed: `unstructured.SetNestedField` does not help because it has the same problem, it writes to a nil map.